neurodamus/node.py

```diff
@@ -455,6 +453,7 @@ def create_cells(self, load_balance=None):
                 logging.warning("Skipped node population (restrict_node_populations)")
                 continue
             self._circuits.new_node_manager(circuit, self._target_manager, self._run_conf,
+                                            load_balancer=load_balance,
```

Comment:
As I understand we run `compute_load_balance` on the first non-virtual population of the circuit, so shall the load_balancer be applied to that population?  
Is it feasible to compute lb for all populations and load them by population?

Reply:
Actually in the currently implementation I'm curious whether it would be able to resume for circuits which loadbal is provided but doesnt contain info for that circuit

Reply:
The last commit should solve the issue. I will start drafting applying loadbalance for all circuits

Reply:
The applying part looks good, thanks for implementing the population information along with the lb files. 
One more question, I see in `compute_load_balance` ( line 345 of the same file), we still compute LB for the first population. Is there a plan to compute for all?

Reply:
Yes, that is a logical next step I'd like to tackle as I have time